### DISCUSSION TITLE
Created row does not appear in table overview

### DISCUSSION THREAD

**Discussion by maeventer** [Budibase/budibase#6479] [Help] [answered]
Hey budibase community,
I´m new to budibase and have an issue when creating a new row in my test table. The request to create the row succeded but my table overview does not show anything anything.
![image](https://user-images.githubusercontent.com/41051029/176123195-bf594b5c-33f9-4180-bd4e-f97ffdda31c2.png)

I´ve installed the latest helm chart of budibase in our kubernetes cluster. Any idea what I´m doing wrong?

**Comment by melohagan** [comment, marked as answer]
Hi @mebo4b 

There appears to be an error on the search endpoint. Could you click on it and screenshot the error response please?

**Reply by maeventer** [reply]
Hi @melohagan 
thanks for your fast response. I got a 504 gateway timeout after 1 minute.
![image](https://user-images.githubusercontent.com/41051029/176164451-5f79025b-2418-4d80-9852-842fbb4cf50f.png)

**Reply by melohagan** [reply]
Would you be able to share the proxy service logs please? It might be called **bbproxy**

The command should roughly be: `kubectl logs -f proxy-service --all-containers=true --namespace budibase`

**Reply by maeventer** [reply]
Sure, it seems to be the client closed the connection. Here is the proxy-service log of failed search request.
```
2022/06/29 05:51:07 [info] 34#34: *730 epoll_wait() reported that client prematurely closed connection, so upstream connection is closed too while sending request to upstream, client: 10.244.130.4, server: _, request: "POST /api/ta_bb82587b709e4b16bdac38cb49a11975/search HTTP/1.1", upstream: "http://10.244.108.16:4002/api/ta_bb82587b709e4b16bdac38cb49a11975/search", host: "budibase-dev.suchtberatung.digital", referrer: "https://budibase-dev.suchtberatung.digital/builder/app/app_dev_8b6c8dbcb29647c68e8507a06a2a0f76/data/table/ta_bb82587b709e4b16bdac38cb49a11975"
10.244.130.4 - - [29/Jun/2022:05:51:07 +0000] "POST /api/ta_bb82587b709e4b16bdac38cb49a11975/search HTTP/1.1" 499 0 "https://budibase-dev.suchtberatung.digital/builder/app/app_dev_8b6c8dbcb29647c68e8507a06a2a0f76/data/table/ta_bb82587b709e4b16bdac38cb49a11975" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/06/29 05:51:07 [info] 34#34: *726 client 10.244.130.4 closed keepalive connection
```
I also had a look at the app-service logs but there is no error log written.

**Reply by maeventer** [reply]
Hi @melohagan 
Maybe to give a bit more context, We use gridscale as kubernetes provider with our already installed nginx ingress controller. Therefore i´ve overwritten the following configs:
```
ingress:
  enabled: true
  nginx: false
  annotations:
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/client-max-body-size: 150M
    nginx.ingress.kubernetes.io/proxy-body-size: 50m
    cert-manager.io/cluster-issuer: letsencrypt-production
  tls:
    - hosts:
        - <domain>
      secretName: <domain>
  hosts:
    - host: <domain>
      paths:
        - path: /
          pathType: Prefix
          backend:
            service:
              name: proxy-service
              port:
                number: 10000
services:
  objectStore:
    storage: 1Gi
  redis:
    storage: 1Gi
```

I´ve also tried to install the chart in my local docker-desktop cluster using the default configs from budibase chart but I have the same error.

**Reply by shogunpurple** [reply]
Hi @mebo4b - we've seen this one happen a few times with a certain version of couchDB. Could you try this:
https://github.com/Budibase/budibase/issues/6168#issuecomment-1154026447

We may need to downgrade our helm couchDB version entirely, as this has occurred with a few users.

**Reply by maeventer** [reply]
Hi @shogunpurple - thanks for the hint. With the old couchdb version it´s working 👍

**Comment by melohagan** [comment]
@mebo4b This may also be related: https://github.com/Budibase/budibase/discussions/6536
